### PR TITLE
Moved fingerprint creation to an event model using a receiver

### LINKED ISSUE #109: Utilize django signals for transforming fact collections into fingerprints
# Opening a new issue
___

## Specify type:
- Enhancement

### Bug severity (if applicable):
- High

## Description:
In researching integration with elasticsearch, I stumbled across django signals.  Ideally, I would have used these originally. 
___

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #109

### CODE REVIEW

File: quipucords/api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# pylint: disable=invalid-name
```
Comment (chambridge) [inline quipucords/api/__init__.py]: Add copyright at top of file, please.

File: quipucords/api/fact_views.py [hunk 2]
```diff
@@ -35,45 +34,3 @@ class FactViewSet(mixins.CreateModelMixin,
     def __init__(self, *args, **kwargs):
         super(FactViewSet, self).__init__(*args, **kwargs)
         self.engine = Engine()
```
Comment (chambridge) [inline quipucords/api/fact_views.py]: Do we need the engine here still?

File: quipucords/api/tests_report.py [hunk 3]
```diff
@@ -44,35 +43,14 @@ def create_fact_collection(self,
...
-        for fingerprint in fingerprints:
-            self.assertTrue(isinstance(fingerprint, SystemFingerprint))
-            self.assertIsInstance(str(fingerprint), str)
-
     def test_get_report_list(self):
         """ Create fact collection object via API."""
 
         url = '/api/v1/reports/'
 
         # Create a system fingerprint
```
Comment (chambridge) [inline quipucords/api/tests_report.py]: seems like a stale comment.

File: quipucords/api/tests_report.py [hunk 4]
```diff
@@ -89,8 +67,8 @@ def test_get_fact_collection_report(self):
...
```
Comment (chambridge) [inline quipucords/api/tests_report.py]: Seems like a stale comment.

File: quipucords/api/tests_report.py [hunk 5]
```diff
@@ -106,8 +84,8 @@ def test_get_fact_collection_404(self):
...
```
Comment (chambridge) [inline quipucords/api/tests_report.py]: seems like a stale comment.

File: quipucords/api/__init__.py (hunk 1 above)
Comment (kholdaway) [reply quipucords/api/__init__.py]: Done.

File: quipucords/api/fact_views.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/api/fact_views.py]: Nope

File: quipucords/api/tests_report.py (hunk 3 above)
Comment (kholdaway) [reply quipucords/api/tests_report.py]: Its not a stale statement but its also not obvious why it is there.  I will update the comment.

**Pull request merged**